transports/httpclient/src/main/java/org/commonjava/maven/galley/transport/htcli/internal/HttpDownload.java

```diff
@@ -128,6 +150,18 @@ public DownloadJob call()
         }
     }
 
+    // Generate a mangled name for metric/honeycomb. e.g., https://maven.apache.org to https.maven_apache_org
+    private String mangledName()
+    {
+        return name( this.request.getProtocolVersion().getProtocol(),
```

Comment:
Does this mean that all request from same host name with same protocol will be collected into same one metrics?

Reply:
Honeycomb data are collected per 'span', similar to per request. And we can query them and make a collective report, e.g., the download latency for a specific remote server.

Reply:
So the short answer to the question is yes.

Reply:
Ok. Then no question here.